docs/marketplace/jenkins/testing-distribution/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Setting Up Appcircle Testing Distribution Plugin
+sidebar_label: Testing Distribution
+description: Enhance powerful plugin to distribute your builds to appcircle
+tags:
+  [
+    testing-distribution,
+    ipa distribution,
+    apk distribution,
+    binary distribution,
+  ]
+sidebar_position: 1
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+The Appcircle Testing Distribution plugin allows users to upload their apps and start distribution to test groups or individuals.
+
+### Install Appcircle Testing Distribution Plugin
+
+Go to your Jenkins dashboard and follow, Manage Jenkins > Plugins and search for "Appcircle Testing Distribution"
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/sp-158-installation_steps.png' />
+
+### Add Plugin in Build Steps
+
+Go to your configuration page of the project add a build step.
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/sp-158-add_pannel.png' />
+
+### Configure Plugin
+
+After adding the plugin into your build steps make sure you fulfill required inputs.
+Also, do not forget to add the plugin after your build steps. Because you will be asked to provide the build path.
```

Comment:
After adding the plugin to your build steps, ensure that you provide all required inputs. Additionally, remember to place the plugin after your build steps. You will need to specify the build path during configuration.